oxidation/libparsec/crates/core_fs/src/storage/manifest_storage.rs

```diff
@@ -92,16 +95,29 @@ pub struct ManifestStorage {
     local_symkey: SecretKey,
     conn: Arc<Mutex<SqliteConn>>,
     pub realm_id: EntryID,
+    pub(crate) caches: Arc<Mutex<HashMap<EntryID, Arc<Mutex<Cache>>>>>,
```

Comment:
Why do you need Arc for the values ?

Reply:
The goal is to not lock the entire map, when manipulating a single value.